app/src/main/res/values/colors.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
     <color name="colorPrimary">#6124e2</color>
     <color name="colorPrimaryDark">#410ab5</color>
-    <color name="colorAccent">#6124e2</color>
+    <color name="colorAccent">#ace5d1</color>
     <color name="purple">#A192FF</color>
     <color name="purple_dark">#7654FF</color>
     <color name="neutral">#555e6f</color>
```

Comment:
Please use the LUK colors. Some of these are from the old / legacy colors, and others are not defined in LUK.